help/policies/code_of_conduct_enforcement.md

```diff
@@ -0,0 +1,39 @@
+# arXiv Code of Conduct Enforcement Options
+
+arXiv is a platform where researchers engage with one another to participate in science, software development, and scholarly communications. It is also a community.
+
+Our valued community of authors, readers, moderators, advisory board members, staff, collaborators, and supporters expects and deserves a respectful environment, which is why we adhere to a [Code of Conduct](code_of_conduct), [moderation standards]( /help/moderation), and other policies. 
+
+There are several actions that if taken, may result in a Code of Conduct violation. Such actions include but are not limited to utilizing inappropriate means of communication, abusive emails directed toward staff, repeatedly submitting inappropriate material, and may occur on any of arXiv’s channels. The Code of Conduct does not address issues concerning the acceptance of research articles. Any issues concerning moderation can utilize the appeals process found at: (/help/moderation). 
+
+If a community member feels that a violation has occurred, the violation can be reported via the [Code of Conduct reporting form](https://cornell.ca1.qualtrics.com/jfe/form/SV_6FgHTrYs5JT30wZ). Please note that any communication received through this form is treated confidentially in keeping with arXiv’s privacy policies. 
+
+A committee formed of a combination of arXiv staff, leadership, and/or advisory board members will then determine whether or not a violation has occurred. 
```

Comment:
Suggestion: change "advisory board members" to "advisors"

Reply:
accepted